### PR TITLE
[4.0] Finder date filters

### PR DESCRIPTION
**Pull request by brianteeman**
Maintain the gap between the two filters

### before
image

### after
image

### CODE REVIEW

**Comment by rjharishabh** [comment]
I have tested this item :white_check_mark: successfully on 1e51ccbabfe4b7ee66f1c0aa65377f06afe9f802<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33544">issues.joomla.org/tracker/joomla-cms/33544</a>.</sub>

File: administrator/components/com_finder/src/Service/HTML/Filter.php [hunk 1]
```diff
@@ -67,7 +67,7 @@ public function slider($options = array())
 			}
 			catch (\RuntimeException $e)
 			{
-				return null;
+				return;
```
Comment (Quy) [inline administrator/components/com_finder/src/Service/HTML/Filter.php]: This requires updating `* @return`.

Comment (brianteeman) [reply administrator/components/com_finder/src/Service/HTML/Filter.php]: weird - I didnt intentionally make this change, must have been done automatically after i set up phpcs-fixer

Comment (brianteeman) [reply administrator/components/com_finder/src/Service/HTML/Filter.php]: I've removed these changes

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 62c65b2086d651f7b2cf054e2a446eca9279178f<br /><br />@rjharishabh can you please retest?<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33544">issues.joomla.org/tracker/joomla-cms/33544</a>.</sub>

**Comment by rjharishabh** [comment]
I have tested this item :white_check_mark: successfully on 62c65b2086d651f7b2cf054e2a446eca9279178f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33544">issues.joomla.org/tracker/joomla-cms/33544</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/33544">issues.joomla.org/tracker/joomla-cms/33544</a>.</sub>

**Comment by Quy** [comment]
Thanks

**Comment by brianteeman** [comment]
thanks

**Pull request merged**